chore(fleets)_: Remove unsupported fleets

friofry opened the pull request:
Important changes:

* removed fleets (to match https://fleets.waku.org/)
  * `status.prod`(FleetStatusProd)
  * `status.test`(FleetStatusTest)
  * `eth.prod`(FleetProd) 
* default fleet `eth.prod` -> `shards.test` 
  * as mentioned https://github.com/status-im/infra-shards/issues/33
* updated fleet.go, bindata.go

Closes #5381
desktop PR https://github.com/status-im/status-desktop/pull/15427

friofry commented on _assets/scripts/update-fleet-config.sh:
```diff
@@ -5,12 +5,12 @@ DIR="$(cd $(dirname "$0")/../../config/cli; pwd)"
 json=$(curl --silent https://fleets.status.im/)
 fleets=(
-    'eth.prod'
+    'shards.test'
```
probably should be removed

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (51)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 40a8cfeb | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/1/) | 2024-07-02 15:34:02 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240702-152858-40a8cf-pr5468.zip) |
| :heavy_check_mark: | 40a8cfeb | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/1/) | 2024-07-02 15:43:41 | ~14 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240702-152858-40a8cf-pr5468.zip) |
| :heavy_check_mark: | 40a8cfeb | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/1/) | 2024-07-02 15:45:54 | ~16 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240702-152858-40a8cf-pr5468.aar) |
| :heavy_multiplication_x: | 40a8cfeb | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/1/) | 2024-07-02 15:52:15 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e8a0c919 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/2/) | 2024-07-03 13:37:25 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240703-133406-e8a0c9-pr5468.zip) |
| :heavy_check_mark: | e8a0c919 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/2/) | 2024-07-03 13:37:51 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240703-133406-e8a0c9-pr5468.zip) |
| :heavy_check_mark: | e8a0c919 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/2/) | 2024-07-03 13:39:27 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240703-133402-e8a0c9-pr5468.aar) |
| :heavy_multiplication_x: | e8a0c919 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/2/) | 2024-07-03 13:45:12 | ~11 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/2//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | f451822e | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/1/) | 2024-07-05 12:40:48 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/1//consoleText) |
| :heavy_check_mark: | f451822e | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/3/) | 2024-07-05 12:40:53 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-123826-f45182-pr5468.aar) |
| :heavy_check_mark: | f451822e | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/3/) | 2024-07-05 12:41:45 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-123826-f45182-pr5468.zip) |
| :heavy_check_mark: | f451822e | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/3/) | 2024-07-05 12:42:19 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-123826-f45182-pr5468.zip) |
| :heavy_multiplication_x: | f451822e | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/3/) | 2024-07-05 12:47:24 | ~8 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/3//consoleText) |
| :heavy_multiplication_x: | f451822e | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/4/) | 2024-07-09 15:53:00 | ~8 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/4//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 72f156dd | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/2/) | 2024-07-10 12:32:18 | ~1 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/2//consoleText) |
| :heavy_check_mark: | 72f156dd | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/4/) | 2024-07-10 12:32:39 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240710-123021-72f156-pr5468.aar) |
| :heavy_check_mark: | 72f156dd | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/4/) | 2024-07-10 12:32:51 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240710-123021-72f156-pr5468.zip) |
| :heavy_check_mark: | 72f156dd | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/4/) | 2024-07-10 12:33:27 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240710-123021-72f156-pr5468.zip) |
| :heavy_multiplication_x: | 72f156dd | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/5/) | 2024-07-10 12:37:10 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/5//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 5fc95dbb | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/3/) | 2024-07-10 12:34:19 | ~1 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/3//consoleText) |
| :heavy_check_mark: | 5fc95dbb | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/5/) | 2024-07-10 12:34:27 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240710-123302-5fc95d-pr5468.aar) |
| :heavy_check_mark: | 5fc95dbb | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/5/) | 2024-07-10 12:35:06 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240710-123308-5fc95d-pr5468.zip) |
| :heavy_check_mark: | 5fc95dbb | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/5/) | 2024-07-10 12:36:28 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240710-123352-5fc95d-pr5468.zip) |
| :heavy_multiplication_x: | 5fc95dbb | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/4/) | 2024-07-10 12:46:52 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/4//consoleText) |
| :heavy_multiplication_x: | 5fc95dbb | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/5/) | 2024-07-10 12:52:01 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/5//consoleText) |
| :heavy_multiplication_x: | 5fc95dbb | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/6/) | 2024-07-10 12:53:38 | ~16 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0940e2e6 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/6/) | 2024-07-11 12:01:01 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240711-115828-0940e2-pr5468.zip) |
| :heavy_multiplication_x: | 0940e2e6 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/6/) | 2024-07-11 12:01:39 | ~3 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/6//consoleText) |
| :heavy_check_mark: | 0940e2e6 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/6/) | 2024-07-11 12:01:51 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240711-115828-0940e2-pr5468.zip) |
| :heavy_check_mark: | 0940e2e6 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/6/) | 2024-07-11 12:04:54 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240711-115828-0940e2-pr5468.aar) |
| :heavy_multiplication_x: | 0940e2e6 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/7/) | 2024-07-11 12:22:21 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/7//consoleText) |
| :heavy_multiplication_x: | 0940e2e6 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/8/) | 2024-07-11 13:24:53 | ~43 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/8//consoleText) |
| :heavy_multiplication_x: | 0940e2e6 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/9/) | 2024-07-11 13:50:17 | ~5 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/9//consoleText) |
| :heavy_multiplication_x: | 0940e2e6 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/10/) | 2024-07-11 14:12:15 | ~15 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/10//consoleText) |
| :heavy_multiplication_x: | 0940e2e6 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/11/) | 2024-07-11 14:53:50 | ~16 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/11//consoleText) |
| :heavy_multiplication_x: | 0940e2e6 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/12/) | 2024-07-11 15:31:09 | ~22 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/12//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | fc9375b2 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/7/) | 2024-07-30 10:15:51 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/7//consoleText) |
| :heavy_check_mark: | fc9375b2 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/7/) | 2024-07-30 10:17:25 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240730-101315-fc9375-pr5468.zip) |
| :heavy_check_mark: | fc9375b2 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/7/) | 2024-07-30 10:17:34 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240730-101315-fc9375-pr5468.zip) |
| :heavy_check_mark: | fc9375b2 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/7/) | 2024-07-30 10:18:43 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240730-101311-fc9375-pr5468.aar) |
| :heavy_multiplication_x: | fc9375b2 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/13/) | 2024-07-30 10:59:04 | ~45 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/13//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 3855f5f2 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/8/) | 2024-07-30 20:19:14 | ~1 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/8//consoleText) |
| :heavy_check_mark: | 3855f5f2 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/8/) | 2024-07-30 20:19:43 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240730-201720-3855f5-pr5468.aar) |
| :heavy_check_mark: | 3855f5f2 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/8/) | 2024-07-30 20:20:56 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240730-201720-3855f5-pr5468.zip) |
| :heavy_check_mark: | 3855f5f2 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/8/) | 2024-07-30 20:21:05 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240730-201720-3855f5-pr5468.zip) |
| :heavy_multiplication_x: | 3855f5f2 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/1/) | 2024-07-30 20:40:25 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/1//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 3258abd8 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/9/) | 2024-08-01 10:37:08 | ~1 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/9//consoleText) |
| :heavy_check_mark: | 3258abd8 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/9/) | 2024-08-01 10:37:35 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240801-103510-3258ab-pr5468.aar) |
| :heavy_check_mark: | 3258abd8 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/9/) | 2024-08-01 10:37:37 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240801-103514-3258ab-pr5468.zip) |
| :heavy_check_mark: | 3258abd8 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/9/) | 2024-08-01 10:38:45 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240801-103510-3258ab-pr5468.zip) |
| :heavy_multiplication_x: | 3258abd8 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/2/) | 2024-08-01 10:59:03 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/2//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 2a5bcdaa | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/10/) | 2024-08-01 10:52:59 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240801-105126-2a5bcd-pr5468.aar) |
| :heavy_multiplication_x: | 2a5bcdaa | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/10/) | 2024-08-01 10:53:24 | ~1 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/10//consoleText) |
| :heavy_check_mark: | 2a5bcdaa | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/10/) | 2024-08-01 10:53:29 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240801-105131-2a5bcd-pr5468.zip) |
| :heavy_check_mark: | 2a5bcdaa | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/10/) | 2024-08-01 10:56:08 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240801-105131-2a5bcd-pr5468.zip) |
| :heavy_multiplication_x: | 2a5bcdaa | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/3/) | 2024-08-01 11:00:23 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/3//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 2fadf455 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/4/) | 2024-08-01 11:28:00 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5468/4//consoleText) |
| :heavy_check_mark: | 2fadf455 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5468/11/) | 2024-08-01 11:28:26 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240801-112646-2fadf4-pr5468.aar) |
| :heavy_check_mark: | 2fadf455 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5468/11/) | 2024-08-01 11:29:04 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240801-112650-2fadf4-pr5468.zip) |
| :heavy_multiplication_x: | 2fadf455 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/11/) | 2024-08-01 11:29:09 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5468/11//consoleText) |
| :heavy_check_mark: | 2fadf455 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5468/11/) | 2024-08-01 11:30:26 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240801-112650-2fadf4-pr5468.zip) |

yakimant reviewed (commented):
`eth.prod` is still online.
I'm not sure how it is beings used. Maybe it's worth to share with Mobile/Desktop guys.

If we don't need it - can we cleanup in other repos and shutdown the servers?

yakimant commented:
FYI @andremedeiros, @anainsomnia, @churik

jrainville commented:
I think you pinged the wrong person. Andre is not at Status anymore :joy: 

cc @cammellos

igor-sirotin commented on protocol/messenger_mailserver_cycle.go:
```diff
@@ -172,7 +172,7 @@ func (m *Messenger) getFleet() (string, error) {
 		fleet = m.config.clusterConfig.Fleet
 	} else {
-		fleet = params.FleetProd
+		fleet = params.FleetShardsTest
```
Probably a good place to reuse this:
https://github.com/status-im/status-go/blob/1ba4b86c7e2a63864aa84698039e7a1d46082701/api/defaults.go#L44

friofry replied:
I have moved DefaultFleet to the params package.

Otherwise, if I use `api.DefaultFleet` here, then there is a cyclyc dependency:

```
package github.com/status-im/status-go/build/bin/statusgo-lib
	imports github.com/status-im/status-go/mobile
	imports github.com/status-im/status-go/api
	imports github.com/status-im/status-go/node
	imports github.com/status-im/status-go/services/accounts
	imports github.com/status-im/status-go/protocol
	imports github.com/status-im/status-go/api: import cycle not allowed
```

igor-sirotin commented on params/config_test.go:
```diff
@@ -20,7 +20,7 @@ func TestNewNodeConfigWithDefaults(t *testing.T) {
 		"/some/data/path",
 		params.GoerliNetworkID,
-		params.WithFleet(params.FleetProd),
+		params.WithFleet(params.FleetShardsTest),
```
same about `DefaultFleet`

igor-sirotin commented on params/config_test.go:
```diff
@@ -34,7 +34,7 @@ func TestNewNodeConfigWithDefaults(t *testing.T) {
 	// assert cluster
 	assert.Equal(t, false, c.NoDiscovery)
-	assert.Equal(t, params.FleetProd, c.ClusterConfig.Fleet)
+	assert.Equal(t, params.FleetShardsTest, c.ClusterConfig.Fleet)
```
same about `DefaultFleet`

jakubgs commented on _assets/scripts/gen_config.sh:
```diff
@@ -10,7 +10,7 @@ LISTEN_PORT="${LISTEN_PORT:-30303}"
 MAX_PEERS="${MAX_PEERS:-50}"
 DAYS_KEPT="${DAYS_KEPT-30}"
-FLEET_NAME="${FLEET_NAME:-eth.prod}"
+FLEET_NAME="${FLEET_NAME:-shards.test}"
```
So we're essentially replacing waku v1 with waku v2?
Can this not wait until we rename `shards.test` to `status.prod`?

jakubgs replied:
It's not a big deal, we na just change it later tho. Not sure how long the renaming will drag.

fryorcraken replied:
I think renaming is pretty much done on fleet side, now? @jakubgs @richard-ramos

jakubgs replied:
I have finished renaming `shards.test` into `status.prod`. We just need to confirm it works as expected.

yakimant commented:
BTW, `eth.prod` is mainly used by Android app
<img width="1360" alt="Screenshot 2024-07-08 at 11 18 39" src="https://github.com/status-im/status-go/assets/1529006/f8e2f5b8-1c36-4d1d-b1b5-74ee836a7738">

churik commented:
Fleet `eth.prod` is used is the default for users on v1.
And as Jakub mentioned, it is mostly Android users.

jakubgs reviewed (commented):
The `shards.test` fleet has already been renamed to `status.prod`, so might as well use the new name in this PR.

And `eth.prod` is old Waku v1 fleet used by legacy app that nobody cares about anymore.

friofry commented:
> The `shards.test` fleet has already been renamed to `status.prod`, so might as well use the new name in this PR.
> 
> And `eth.prod` is old Waku v1 fleet used by legacy app that nobody care about anymore.

I'll rebase and address it on Monday

friofry commented:
> > The `shards.test` fleet has already been renamed to `status.prod`, so might as well use the new name in this PR.
> > And `eth.prod` is old Waku v1 fleet used by legacy app that nobody care about anymore.
> 
> I'll rebase and address it on Monday

I suggested to @richard-ramos to merge his PR first https://github.com/status-im/status-go/pull/5571, 
and I'll update this PR

friofry committed:
chore(fleets)_: Remove unsupported fleets

* eth.prod
* status.staging

chore(fleets)_: reuse api.DefaultFleet in GetFleet()

fixes #5381

rebased on chore_: rename shards.test to status.prod (#5571)

friofry commented:
rebased 🪜

friofry committed:
chore(fleets)_: add exchangeBootNode to connect alice and bob in tests

friofry commented:
update:

I've removed the `eth.prod` fleet. And `SyncDeviceSuite` started failing `TestPairAcceptContactRequest`. It tries to receive a `receiveContactRequest` message with no luck (bob sends to alice1)
 I noticed that the ClusterConfigs look quite different.

@richard-ramos suggested to make sure that nodes are being created are connected to each other
@qfrank suggested to add `exchangeBootNode`, similar to https://github.com/status-im/status-go/blob/daef6ee030e98f1a778762daab18945ad3cdb496/api/messenger_raw_message_resend_test.go#L41

jakubgs commented:
Maybe it's time to merge this.

churik commented:
Will it affect all users that remained on 1.20.6 app? 

Also in order to merge it, please, rebase it with a recent changes, thannks

jakubgs commented:
>Will it affect all users that remained on 1.20.6 app?

I don't see why it would, unless we release a hotfix for that v1 release.

churik commented:
@friofry 
can you please resolve the conflicts and we'll test this PR and create corresponding PR in status-mobile? 
Seems it got lost.

igor-sirotin commented:
@friofry @churik I think this PR doesn't make sense anymore, I'm moving the fleets around in https://github.com/status-im/status-go/pull/6414. And `eth.prod` is not there already.
